omod/src/main/webapp/resources/scripts/conditionlist/controllers/editcondition.controller.js

```diff
@@ -0,0 +1,141 @@
+var app = angular.module("conditionApp",
+    ['uicommons.widget.coded-or-free-text-answer', 'app.restfulServices', 'app.models', 'app.commonFunctionsFactory']);
+
+app.controller("ConditionController", ConditionController);
+ConditionController.$inject = ['$scope', 'RestfulService', 'ConditionModel', 'ConceptModel', 'CommonFunctions'];
+
+function ConditionController($scope, RestfulService, ConditionModel, ConceptModel, CommonFunctions) {
+    var self = this;
+
+    $scope.patientUuid = null;
+    $scope.condition = null;
+    var inactiveStatus = 'INACTIVE';
```

Comment:
Are you planning to change the value of inactiveStatus?

Reply:
No

Reply:
Then why isn't a constant?